apps/studio/public/assets/css/preview-tw.css

```diff
@@ -4133,11 +4129,29 @@ video {
   border-color: rgb(247 143 30 / var(--tw-border-opacity));
 }
 
+.focus\:bg-utility-highlight:where([data-rac])[data-focused] {
```

Comment:
are these css changes related to this page order?

Reply:
No, I think might have been previous changes that went in without doing the build step.